File: nekoshlink/shlink-security/src/main/java/org/nekosoft/shlink/sec/audit/GenericAuthenticationFailureEvent.java
package org.nekosoft.shlink.sec.audit;

import org.springframework.security.authentication.event.AbstractAuthenticationFailureEvent;
import org.springframework.security.core.Authentication;
import org.springframework.security.core.AuthenticationException;

public class GenericAuthenticationFailureEvent extends AbstractAuthenticationFailureEvent {
    public GenericAuthenticationFailureEvent(Authentication auth, AuthenticationException authEx) {
        super(auth, authEx);
    }
}

File: nekoshlink/shlink-security/src/main/kotlin/org/nekosoft/shlink/sec/audit/GenericAuthenticationFailureEvent.kt
package org.nekosoft.shlink.sec.audit

import org.springframework.security.authentication.event.AbstractAuthenticationFailureEvent
import org.springframework.security.core.Authentication
import org.springframework.security.core.AuthenticationException

class GenericAuthenticationFailureEvent(
    auth: Authentication,
    authEx: AuthenticationException,
) : AbstractAuthenticationFailureEvent(auth, authEx)
